Answer Key for Examples.Rmd

```diff
@@ -7,18 +7,35 @@ output: html_document
 
 # Questions
 
-1. You just got your BB gun for Christmas but all is for naught because an astroid is bearing down on Earth! The astroid is $1  \times 10^{15} kg$ and moving at $2 \times 10^{4} m/s$. Fortunatly you have quite a few BBs which weigh $0.0002 kg$ and you can fire them at $60 m/s$. How many BBs will it take to stop the astroid and save Earth?
```

Comment:
Please keep your previous problem sets around on the master branch. Maybe a new filename rather than deleting the old answer key?